.github/workflows/macos.yml

```
@@ -46,11 +46,13 @@ jobs:
         uses: ammaraskar/gcc-problem-matcher@0.2.0
 
       - name: Build (${{ matrix.build-type }})
+        env:
+            CMAKE_PREFIX_PATH: ${{ matrix.qt-version-major == '6'}} && ${{ env.Qt6_Dir }} || ${{ env.Qt5_Dir }}
         run: |
           mkdir build
           cd build
-          qmake -config ${{ matrix.build-type }} ../src/Notes.pro
-          make -j$(sysctl -n hw.logicalcpu)
+          cmake .. -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -DCMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH}"
```

```suggestion
          cmake . -B build-dir -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -DCMAKE_PREFIX_PATH="$(brew --prefix qt@${{ matrix.qt-version-major }})"
```

Then the `mkdir` and `cd` lines can be removed. Just make sure to also update the path of the build directory in subsequent build steps.

Also, It was not necessary to use `-DCMAKE_PREFIX_PATH` on my macOS 11 VM, but for whatever reason we have to do this on GitHub Actions.

---

I had to add -DCMAKE_PREFIX_PATH because Qt gets installed in the `  cwd/Qt`   folder or something, so it doesn't find it by default.

---

We're in the homebrew step, so they install it to `/opt/homebrew`, IIRC. It's `aqtinstall` that installs Qt on the current directory...

In any case, setting it to `-DCMAKE_PREFIX_PATH="$(brew --prefix qt@${{ matrix.qt-version-major }})"` should work...